client/src/components/TestSuite/TestSuiteDetails/TestGroupCard.tsx

```diff
@@ -20,29 +20,32 @@ const TestGroupCard: FC<TestGroupCardProps> = ({
   const styles = useStyles();
 
   const runnableType = 'tests' in runnable ? RunnableType.TestGroup : RunnableType.TestSuite;
-
+  const showRunButton = runnableIsTestSuite(runnable) || (runnable as TestGroup).user_runnable;
+  const runButton = showRunButton ? (
+    <Tooltip title={testRunInProgress ? 'Disabled - Ongoing Test.' : ''} arrow>
```

Comment:
Can we pull these run buttons out into their own component?

Reply:
Moved them into their own component.  In doing so, I think I see other ways this could be organized to reduce how much stuff needs to be passed between levels.  However, I think at this point we shouldn't continue to refactor until I spend some time moving stuff around in the interface a bit, I have a feeling we will want to change some of this stuff at a more fundamental level, and refactoring more here doesn't help us with that.